1-mandatory/1-writers.js

```diff
@@ -61,8 +66,20 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+function getDeadWritersInTheirForties(writer) {
+  return writer.age > 40 && writer.age < 49 && writer.alive === false;
```

Comment:
Issue: This check will exclude people of age 40 or 49, but they are still in their 40s